get rid of weird line breaks and use action

linkslice opened the pull request:
- Link the PR to an issue for new features
- Rebase your PR if it gets out of sync with master
  When using restarts/stops/starts you'd get weird output sometimes, Strange line breaks 'OK' status overlapping the next lines etc...
  This fixes those moves OK to the right place and colorizes them correctly.

linkslice committed:
get rid of weird line breaks and use action

When using restarts/stops/starts you'd get weird output sometimes, Strange line breaks 'OK' status overlapping the next lines etc...
This fixes those moves OK to the right place and colorizes them correctly.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=4926) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=4926) before we can accept your contribution.<br/>

linkslice committed:
added : for more uniformity

when doing a restart the output looked like:
Stopping Grafana Server ...                                [  OK  ]
Starting Grafana Server: ...                               [  OK  ]

The Stopping line did not have a colon. I added it just to make it look better.

torkelo commented on packaging/rpm/init.d/grafana-server:
```diff
@@ -90,7 +88,7 @@ case "$1" in
     # Start Daemon
     cd $GRAFANA_HOME
-    su -s /bin/sh -c "nohup ${DAEMON} ${DAEMON_OPTS} >> /dev/null 3>&1 &" $GRAFANA_USER 2> /dev/null
+    action $"Starting $DESC: ..." su -s /bin/sh -c "nohup ${DAEMON} ${DAEMON_OPTS} >> /dev/null 3>&1 &" $GRAFANA_USER 2> /dev/null
```
is action a universal thing you can use? Does it work on CentOS, Fedora, OpenSuse, RedHat?

linkslice replied:
Action is a function from init.d. It'll be everywhere that is.

torkelo replied:
dammit, my fears where justified :( 
seems action is not available in common distro versions :(

https://github.com/grafana/grafana/issues/6968
